dan_layer/storage/src/consensus_models/block.rs

```diff
@@ -96,6 +96,9 @@ pub struct Block {
     /// Signature of block by the proposer.
     #[cfg_attr(feature = "ts", ts(type = "{public_nonce : string, signature: string} | null"))]
     signature: Option<ValidatorSchnorrSignature>,
+    block_time: Option<u64>,
```

Comment:
block_time should not be in the block that is sent between peers, over the wire, though it can still be calculated when inserting the block in the database and sent via JRPC.